.pre-commit-config.yaml

```diff
@@ -55,3 +48,14 @@ repos:
       hooks:
           - id: prettier
             exclude: templates
+    - repo: https://github.com/pycqa/isort
+      rev: 5.8.0
+      hooks:
+          - id: isort
+            name: isort (python)
+          - id: isort
+            name: isort (cython)
+            types: [cython]
+          - id: isort
+            name: isort (pyi)
+            types: [pyi]
```

Comment:
`exclude: templates` here as well?

Reply:
Wasn't sure. Actually don't think that it's wrong to also apply it to the templates in this case. Eventually a pre-commit setup for the templates would be cool anyways #290